src/components/ChattingInput/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { ChangeEvent } from "react";
+import * as S from "./style";
+import SendIcon from "../../Assets/svg/SendIcon";
+
+interface Props {
+  inputValue: string;
+  setInputValue: React.Dispatch<React.SetStateAction<string>>;
+}
+
+const ChattingInput: React.FC<Props> = ({ inputValue, setInputValue }) => {
+  const handleInputChange = (e: ChangeEvent<HTMLInputElement>) =>
+    setInputValue(e.target.value);
+
+  const handleSendClick = () => {
+    // 요청 보내기
+  };
+
+  return (
+    <S.ChattingInput>
+      <S.InputContainer>
+        <S.Input value={inputValue} onChange={handleInputChange} />
+        <S.SendButton onClick={handleSendClick}>
+          <SendIcon />
+        </S.SendButton>
+      </S.InputContainer>
+    </S.ChattingInput>
```

Comment:
```suggestion
    <S.ChattingInput>
      <S.InputContainer onSubmit={handleSendClick}>
        <S.Input value={inputValue} onChange={handleInputChange} />
        <S.SendButton>
          <SendIcon />
        </S.SendButton>
      </S.InputContainer>
    </S.ChattingInput>
```

form 테그 변경 후 `onSubmit` 핸들러를 사용해서 입력 완료를 처리하는 것이 좋아보입니다.

이렇게 하면 입력 후 엔터를 눌렀을때 메시지가 보내질 수 있을것 같아요

Reply:
좋은 의견인거 같습니다
[4942883](https://github.com/ONEUS-team/gsmatch-front/pull/21/commits/4942883008d20c3e3f5bd4b660b2060bd0490986)